docs/configure.md

```diff
@@ -59,21 +59,21 @@ label: my_import
 | `archive_rotation`    | ➖                   | integer | `archive.rotation`    | Number of archiving files to be kept   |
 | `quarantine_rotation` | ➖                   | integer | `quarantine.rotation` | Number of quarantine files to be kept  |
 
-### Options inutilisées
+### Unused options
 
 | Name    | Mandatory (default)   | Type    |     |
 |---------|-----------------------| ------- | --- |
 | `label` | ➖ (`null`)            | ?string |     |
 
-### archive
+### Archive
```

Comment:
I kept the option's name case in the title, that's why it was in lowercase.
Your call :+1:

Reply:
Fair enough